lib/hammer_cli_foreman.rb

```diff
@@ -60,15 +60,6 @@ def self.exception_handler_class
       'HammerCLIForeman::Domain', 'hammer_cli_foreman/domain'
     )
 
-    HammerCLI::MainCommand.lazy_subcommand('environment', _("Manipulate environments"),
-      'HammerCLIForeman::PuppetEnvironment', 'hammer_cli_foreman/puppet_environment',
-      :warning => _('%{env} command is deprecated and will be removed in one of the future versions. Please use %{puppet_env} command instead.') % {:env => 'environment', :puppet_env => 'puppet-environment'}
-    )
-
-    HammerCLI::MainCommand.lazy_subcommand('puppet-environment', _("Manipulate Puppet environments"),
-      'HammerCLIForeman::PuppetEnvironment', 'hammer_cli_foreman/puppet_environment'
-    )
-
```

Comment:
Hi everyone, I have joined @amirfefer on removing the puppet-related code from hammer-cli-foreman and moving it to a separate plugin. @adiabramovitch thanks for the related work! 

I think that the `puppet-environment` subcommand should be removed in a follow-up PR for the puppet extraction, not as part of the environment deprecation removal.